packages/neuron-wallet/src/transaction.ts

```diff
@@ -9,19 +10,121 @@ export interface Transaction {
   outputs?: Cell[]
 }
 
-// if addresses set to null, should be all transactions
-export const getTransactions = async (addresses: string[] | undefined, page: number, perPage: number) => {
+export interface Input {
+  previousOutput: OutPoint
+  unlock: Script
+}
+
+const MIN_CELL_CAPACITY: string = '40'
+
+// TODO: should load key from keystore
+// const mockedPrivKey: string = 'e79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3'
+const mockedPublicKey: string = '024a501efd328e062c8675f2365970728c859c592beeefd6be8ead3d901330bc01'
+const mockedAddress: string = '0xbada2cdf5bd644508a0892d17e3c9fba860382a036b600218d20fcdb2e3e816d'
+
+// the system script cell hash
+const systemScriptCellHash = '0x828d1e109a79964521bf5fbbedb4f6e695a9c4b6b674a58887f30c7398e93a76'
+const systemScriptOutPoint: OutPoint = {
+  hash: '',
+  index: 0,
+}
+
+// if addresses set to undefined, should be all transactions
+/* eslint @typescript-eslint/no-unused-vars: "warn" */
+export const getTransactions = async (page: number, perPage: number, _addresses?: string[] | undefined) => {
   const transaction: Transaction = mockedTransaction
 
   return {
     totalCount: page * perPage,
     transactions: Array.from({
       length: perPage,
     }).map(() => transaction),
-    addresses,
   }
 }
 
+const verifyScriptJsonObject = () => {
```

Comment:
This name comes with ruby-sdk, but not proper in JS, we can just name it `verifyScript`. Let's update it in the future.